update_bench_traces_rel.rb

```diff
@@ -2,7 +2,8 @@
 require 'fileutils'
 Dir.chdir(File.dirname(__FILE__))
 goblint = File.expand_path("../analyzer/goblint")
-goblint_conf = File.expand_path("../analyzer/conf/traces-rel.json")
+# goblint_conf = File.expand_path("../analyzer/conf/traces-rel.json")
+goblint_conf = File.expand_path("../analyzer/conf/traces-rel-toy.json")
```

Comment:
This includes some changes to the relational traces script, I'm not sure if we want to change the script or make another copy of the script...

Reply:
I copied it into a separate script such that this could be quickly merged.